connect.js

```diff
@@ -83,6 +83,58 @@ connect.updateUser = function (userId, username, email, phone) {
     });
 }
 
-// Rider functions
+// Rider functions //
+// Gets all current riders
+connect.getRider = function () {
+    con.query("SELECT * FROM `Rider`", function (err, rows) {
+        if (err) throw err;
+        console.log('Riders:\n');
+        console.log(rows); 
+        return rows;
+    });
+}
+
+// Gets rider given id
+connect.getRider = function (riderId) {
+    con.query("SELECT * FROM `Rider` WHERE id = ?", [riderId], function (err, rows) {
+        if (err) throw err;
+        console.log('Rider:\n');
+        console.log(rows); 
+        return rows;
+    });
+}
+
+connect.addRider = function (userId, leaveEarliest, leaveLatest, startId, endPoints) {
```

Comment:
-- first check if bobbytables is a rider. if he is return an empty set
-- takes a user id 
-- select * from Rider where 'input_id' = id